### ISSUE TITLE
Script Generator: Repository structure

### ISSUE THREAD

**Issue by JamesKingWork** [ISISComputingGroup/IBEX#5937]
As an instrument scientist supplying script definitions, I would like to be able to categorise and structure my script definitions. For instance, I would like a folder only for EMU script definitions or a shared set of reflectometry script definitions.

Acceptance criteria

- Set up a structure for the script definitions currently in the repository e.g. putting emu specific definitions into their own folder
- Update in user manual and developer manual the standard for adding script definitions to the repository
- The script generator no longer believes that script definitions are in a flat file system but are in a directory structure
- Create plan/put in a task to update script definitions on instrument to use this new structure liaising with scientists to ensure they can still access their script definitions (they will need the updated script generator before we can do this)

**Comment by JamesKingWork** [comment]
Ticket created as outcome of #5754

**Comment by JamesKingWork** [comment]
Duplicate of https://github.com/ISISComputingGroup/IBEX/issues/5495
